app/controllers/api/v1/accounts/relationships_controller.rb

```diff
@@ -5,7 +5,7 @@ class Api::V1::Accounts::RelationshipsController < Api::BaseController
   before_action :require_user!
 
   def index
-    @accounts = Account.where(id: account_ids).select('id')
+    @accounts = Account.where(id: account_ids).select(:id, :domain)
```

Comment:
Any benefit to also limiting what we select on the single-account cases here, or is that benefit too marginal to bother?

Reply:
I think it's really marginal.